gradle/versions.gradle

```diff
@@ -14,6 +14,7 @@ ext.libVersions =
     commons_lang3: '3.1',
     commons_io: '2.1',
     commons_dbutils: '1.3',
+    dagger: '2.4',
```

Comment:
Supposedly it's required because of breaking changes between dagger 2.0.2 and 2.4. I'm hoping that we can find a way around bumping this because:
 - what if a different product is using 2.0.2?
 - What happens when a product upgrades from dagger 2.4 to dagger 2.X where X has another breaking change?